private_set_intersection/cpp/psi_server.h

```diff
@@ -67,7 +69,8 @@ class PsiServer {
   // Returns INTERNAL if encryption fails.
   StatusOr<psi_proto::ServerSetup> CreateSetupMessage(
       double fpr, int64_t num_client_inputs,
-      absl::Span<const std::string> inputs) const;
+      absl::Span<const std::string> inputs,
+      DataStructure ds = DataStructure::GCS) const;
```

Comment:
Should we make the default `BloomFilter` instead?
```suggestion
      DataStructure ds = DataStructure::BloomFilter) const;
```